apache-couchdb-mixin/dashboards/couchdb-nodes.libsonnet

```diff
@@ -0,0 +1,1268 @@
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 's',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+};
+
+local bulkRequestsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'rate(couchdb_httpd_bulk_requests_total{' + matcher + '}[$__rate_interval])',
+      datasource=promDatasource,
+      legendFormat='cluster="{{cluster}}",instance="{{instance}}"',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Bulk requests',
+  description: 'The number of bulk requests for a node.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 0,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 'reqps',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+};
+
+local responseStatusOverviewPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum by(instance, cluster) (increase(couchdb_httpd_status_codes{' + matcher + ', code=~"2.*"}[$__rate_interval]))',
+      datasource=promDatasource,
+      legendFormat='cluster="{{cluster}}",instance="{{instance}}",status="2xx"',
```

Comment:
Recommend removing 2xx and 3xx

Reply:
Yep, let's wait and see what Gloria's opinion is, because I can definitely see benefits of either option

Reply:
Just as a note, as of now the change has been made to remove the 2xx and 3xx from the pie chart

Reply:
Hmm yes, it's tricky. I see both points with their benefits and shortcomings. I think I do prefer Stefan's idea. Just for having it a nice first visual hint. I think it's fine because we would alert if error rates are too high and also have the panel below to show more details on errors.

Reply:
Great, let's do that then! 😄 
I assume for both this and the pie chart in overview?